### PR TITLE
Finish the new config system

### PR DESCRIPTION
**Pull request by Jason2605**
<Please review the guidelines for contributing to this repository. The link is above.>

Resolves #134.

#### Changes proposed in this pull request: 
- Overhaul of the config system
- Any value which returns a bool can now be used

### CODE REVIEW

**Comment by tmoitoi** [comment]
Problem with clothing, they're invisible and if they've a "" name, they're not showed.

Impossible to buy anything.

**Comment by BoGuu** [comment]
The changes in this PR should address that @tmoitoi

File: Altis_Life.Altis/dialog/function/fn_spawnPointCfg.sqf [hunk 1]
```diff
@@ -79,3 +51,5 @@ if (playerSide isEqualTo civilian) then {
 };
 
 _return;
+
+
```
Comment (Jawshy) [inline Altis_Life.Altis/dialog/function/fn_spawnPointCfg.sqf]: D:

File: Altis_Life.Altis/core/shops/fn_levelCheck.sqf [hunk 2]
```diff
@@ -1,40 +1,38 @@
...
+    if (_evaluation) then {
         _return = true;
     };
 } else {
     _return = true;
 };
 
 _return;
+
+
```
Comment (Jawshy) [inline Altis_Life.Altis/core/shops/fn_levelCheck.sqf]: D:

**Comment by tmoitoi** [comment]
I mean the clothing only, when i've done your last changes all work well except clothings, they're invisible and the others without name will not be shown to the select menu.

**Comment by BoGuu** [comment]
@tmoitoi 

Can you check with the latest change? There is a slight error

**Comment by DardoTheMaster** [comment]
What about integrating playerSkins with config_clothing ? This will allow to set a texture to specific clothes individually.

**Comment by Jason2605** [comment]
What do you mean, this can be done in the playerSkins already? uniform player isEqualTo "U_C_Poloshirt_blue"... etc

**Comment by BoGuu** [comment]
@DardoTheMaster 

Do you mean like the vehicle skins?

**Comment by tmoitoi** [comment]
It's OK now @BoGuu it was me on discord ;)

**Comment by DardoTheMaster** [comment]
@BoGuu
Exactly

**Comment by BoGuu** [comment]
This needs testing, then we can merge 👍

**Comment by setoy** [comment]
The [descriptions](https://github.com/AsYetUntitled/Framework/blob/c0590e5c9c51b52f28be8132714a595c05e6b162/Altis_Life.Altis/config/Config_Clothing.hpp#L6-L9) haven't been updated.

**Pull request merged**